src/components/atoms/SpacesDropdown/SpacesDropdown.scss

```diff
@@ -0,0 +1,60 @@
+@import "scss/constants";
+
+$dropdown-item-icon-size: 20px;
+
+.SpacesDropdown {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &__value {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    font-size: $font-size--lg;
+    font-weight: $font-weight--700;
+    line-height: 19px;
+  }
+
+  &__item {
+    padding: $spacing--sm $spacing--md;
+
+    font-weight: $font-weight--400;
+    font-size: $font-size--md;
+    line-height: 17px;
+
+    &:hover {
+      background: opaque-black(0.05);
+    }
+
+    &-icon {
+      width: $dropdown-item-icon-size;
+      height: $dropdown-item-icon-size;
+      margin-right: $spacing--sm;
+    }
+  }
+
+  .dropdown-menu {
+    background: $room-card-background;
+    box-shadow: 0px 10px 20px opaque-black(0.25);
+    border-radius: $border-radius--lg;
+  }
+
+  .dropdown-toggle.btn {
+    display: flex;
+    align-items: center;
+    padding: $spacing--md $spacing--xl;
+
+    font-size: $font-size--lg;
+    font-weight: $font-weight--500;
+    line-height: 19px;
+
+    &:focus,
+    &:hover {
+      background: opaque-white(0.2);
+      box-shadow: 0px 2px 4px opaque-black(0.25);
+      border-radius: $border-radius--xl;
+    }
```

Comment:
Are these necessary?
Are they meant to modify Bootstrap classes?
Can't just override with different class name like BEM one starting with `&__`?

Reply:
I've overrided .dropdown-toggle and .dropdown-item via BEM class name and left .dropdown-menu. Can't find the best way to modify it with specifity

Reply:
Yeah, it's `globals.scss` that's part of the issue, Bootstrap the core of it.

Reply:
We'll just have to leave it as such while there are Bootstrap components embedded inside `SpacesDropdown`.
The good part is it is isolated to this, not global.